services/spar/test-scim-suite/update.jq

```diff
@@ -0,0 +1,135 @@
+    }
+  }
+]
+
+# in the integration the services run in http
+| (.requests | .[] | .url) |= sub("https"; "http")
+
+# Remove folder: Get Token
+| del(.requests | .[] | select (.folder == "8ea7d081-236a-4cee-af1c-033fb0c4de8b"))
+| del(.folders[] | select (.id == "8ea7d081-236a-4cee-af1c-033fb0c4de8b"))
+| del(.folders_order[] | select (. == "8ea7d081-236a-4cee-af1c-033fb0c4de8b"))
+
+# Remove folder: Group tests
+| del(.requests[] | select (.folder == "d8473c2e-4192-4652-ba3d-e26a6eceb94d"))
+| del(.folders[] | select (.id == "(d8473c2e-4192-4652-ba3d-e26a6eceb94d"))
+| del(.folders_order[] | select (. == "d8473c2e-4192-4652-ba3d-e26a6eceb94d"))
+
+# Remove folder: ComplexAttribute tests
+| del(.requests[] | select (.folder == "23b40117-aa00-4b91-acb3-53f470cd5d83"))
+| del(.folders[] | select (.id == "23b40117-aa00-4b91-acb3-53f470cd5d83"))
+| del(.folders_order[] | select (. == "23b40117-aa00-4b91-acb3-53f470cd5d83"))
+
+# Remove folder: Group tests with garbage
+| del(.requests[] | select (.folder == "53bd1992-2f7d-4dce-9fc1-60ca0d031345"))
+| del(.folders[] | select (.id == "53bd1992-2f7d-4dce-9fc1-60ca0d031345"))
+| del(.folders_order[] | select (. == "53bd1992-2f7d-4dce-9fc1-60ca0d031345"))
+
+# Update Folder "User tests"
+# Post User # FUTUREWORK: use SAML
+| (.requests[] | select (.name == "Post User") | .rawModeData) |= sub("\\$\\{__UUID\\}"; "{{user1externalId}}")
+| (.requests[] | select (.name == "Post User") | .rawModeData) |= sub("UserName123"; "{{user1userName}}")
+# Post Enterprise User
+| (.requests[] | select (.name == "Post EnterpriseUser") | .rawModeData) |= sub("\\$\\{__UUID\\}"; "{{user2externalId}}")
+| (.requests[] | select (.name == "Post EnterpriseUser") | .rawModeData) |= sub("UserName222"; "{{user2userName}}")
+# Get User Attributes
+# Remove this step because we dont support the standard here
+| del(.requests[] | select (.name == "Get User Attributes"))
+# Get User Filters
+# Remove this step because we dont support the standard here
+| del(.requests[] | select (.name == "Get User Filters"))
+# User 2 replace test
+| (.requests[] | select (.name == "User 2 replace test") | .rawModeData) |= sub("\\$\\{__UUID\\}"; "{{user2externalId2}}")
+| (.requests[] | select (.name == "User 2 replace test") | .rawModeData) |= sub("\\$\\{__UUID\\}"; "{{user2externalId2}}")
+# Get user2 Check replace
+#  Remove this step. This test tries to access .name.formatted in the response,
+#  which wire doesn't support. 'name' is not a required attribute (see https://tools.ietf.org/html/rfc7643#section-4.1.1)
+| del(.requests[] | select (.name == "Get user2 Check replace"))
+
+
+# Update Folder "User tests with garbage"
+
+# Step: Post user "OMalley"
+# Use email instead of uuid
+| (.requests[] | select (.name == "Post user \"OMalley\"") | .rawModeData) |= sub("22fbc523-6032-4c5f-939d-5d4850cf3e52"; "{{garbage1_externalId}}")
+# Remove this step because the assertion fails: expected 'omalley' to deeply equal 'OMalley'
```

Comment:
```suggestion
# Remove this step because the assertion fails: expected 'omalley' to deeply equal 'OMalley' -- TODO: fix this?
```

Can you go through the patches again and see if any of them look like a bug in wire to you?  (Does this particular one?)

You don't need to be 100% confident on your assesments, just use your best jugement.

Reply:
yes, it also looks like a bug to me